### ISSUE TITLE
TTC font support?

### ISSUE THREAD

**Issue by ScalletaZ** [vurtun/nuklear#818]
I got crashed on nk_font_atlas_add_from_file with a ttc font.

Is ttc font supported?

**Comment by irtusb** [comment]
no. that function is meant to work with TTF only. If you really want to use that API you'd need to reimplement nk_tt_InitFont [(called here)](https://github.com/vurtun/nuklear/blob/509c75b086351e82865f26a507235b60a63e1538/src/nuklear_font.c#L2208).
The other way you have is not to bake your font at all (render glyphs to an in memory image) and use  `struct nk_user_font`
